includes/class-newspack-popups.php

```diff
@@ -71,6 +71,7 @@ public function __construct() {
 			add_action( 'init', [ __CLASS__, 'register_cpt' ] );
 			add_action( 'init', [ __CLASS__, 'register_meta' ] );
 			add_action( 'init', [ __CLASS__, 'register_taxonomy' ] );
+			add_action( 'init', [ __CLASS__, 'disable_prompts_for_protected_pages' ] );
```

Comment:
Non-blocking: could be hooked to a less broad hook to not fire so often without need. `template_redirect` should be an early enough hook